creator-node/docker-compose.yml

```diff
@@ -20,6 +28,13 @@ services:
       service: base-cache
     networks:
       - creator-node-network
+    logging:
+      options:
+        max-size: 10m
+        max-file: 3
+        mode: non-blocking
+        max-buffer-size: 4m
+      driver: json-file
 
```

Comment:
noting - max-size = 10m and max-file = 3 seem to be the default, same with `json-file` driver